azure-pipelines.yml

```diff
@@ -110,8 +106,8 @@ jobs:
     - script: sh -x ./clean_build.sh
       displayName: Clean Build
     - task: PublishBuildArtifacts@1
-      displayName: Publish Release DMG
-      condition: eq(variables['B_BUILD_TYPE'], 'Release')
+      displayName: Publish Release DMG if on Big Sur agent
+      condition: eq(variables['imageName'], 'macOS-11')
       inputs:
         pathtoPublish: build/bundle
-        artifactName: Mac Release Disk Image and App $(imageName)
+        artifactName: Mac Release DMG and App $(imageName)
```

Comment:
Separate commit (maybe combine with the one which renames `macOS builds`)

Reply:
Fixed and combined with ea38ae65.